konfera/templates/konfera/event_speakers.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'konfera/base.html' %}
+
+{% block title %}<h1>{{ event }} Speakers</h1>{% endblock %}
```

Comment:
@ZoltanOnody yes, `{% block title}` is meant to be the page/tab header. I hope the next commit will make this clear.
And by the way - thank you guys for the comments and patience :)